### DISCUSSION TITLE
Database adapter for Tortoise orm

### DISCUSSION THREAD

**Discussion by Tobi-De** [fastapi-users/fastapi-users#986] [Show and tell]
Hi, I hope everyone is doing well.

With the release of  fastapi-users v10, tortoise orm support was dropped, so I wrote a new database adapter for v10 and beyond that you can find [here](https://github.com/Tobi-De/fastapi-users-tortoise). It is not 100% complete as of this writing but it is usable and tested. I highly recommend anyone with an unsupported database to build their own adapter. @frankie567  made building one quite simple, the old and new ones can be used as inspiration and there is a [cookiecutter](https://github.com/fastapi-users/fastapi-users-db-cookiecutter) available to start the work.

**Comment by frankie567** [comment]
Thank you @Tobi-De 🙏 That's really appreciated ❤️
